examples/src/main/kotlin/com/xebia/functional/xef/evaluator/TestExample.kt

```diff
@@ -48,6 +48,23 @@ object TestExample {
           +OutputResponse(description = fakeOutputs, value = "The movie is Jurassic Park")
         }
       }
-    spec.evaluate<AnswerAccuracy>(success = listOf(AnswerAccuracy.yes))
+    val results = spec.evaluate<AnswerAccuracy>(success = listOf(AnswerAccuracy.yes))
```

Comment:
This is only a way to show the information of the tests in the console, it's not related with the problems with the `Conversation` but we are using it for testing